pandas/tests/series/indexing/test_setitem.py

```diff
@@ -93,6 +93,22 @@ def test_setitem_negative_out_of_bounds(self):
         with pytest.raises(IndexError, match=msg):
             ser[-11] = "foo"
 
+    @pytest.mark.parametrize("ser_index", [0, 1])
```

Comment:
can you parameterize over `tm.loc` and `tm.at` (i assume that `.iloc` / `.iat` take a different path)?

Reply:
Thanks, did not know that. Yes they do not align